ui/src/dashboards/components/TemplateControlBar.js

```diff
@@ -74,4 +112,4 @@ TemplateControlBar.propTypes = {
   isOpen: bool,
 }
 
-export default TemplateControlBar
+export default ControlBarWrapper
```

Comment:
Oh, nvm - I didn't realize that the default export could be given an arbitrary name.